requirements.txt

```diff
@@ -10,11 +10,17 @@ idna==2.6
 itsdangerous==0.24
 Jinja2==2.9.6
 MarkupSafe==1.0
+nested-lookup==0.1.3
+prompt-toolkit==1.0.15
 PyJWT==1.5.3
+PySocks==1.6.7
 pytz==2017.2
 requests==2.18.4
 requests-aws4auth==0.9
 six==1.11.0
+slack-logger==0.2.0
```

Comment:
I don't think this is the same `slack-logger` that you've worked on here - https://github.com/CodeForAfricaLabs/python-slack-logger